net/sslscan/Portfile

```diff
@@ -35,3 +35,5 @@ build.args      PREFIX="${prefix}" \
                 LDFLAGS="[get_canonical_archflags ld] -L${prefix}/lib"
 
 destroot.args   PREFIX="${prefix}"
+
+github.tarball_from archive
```

Comment:
Hm, what is the reason for this change? The default 'tarball' seems fine to me?

Reply:
The `tarball` URL is actually legacy, and though Github still supports it, it's considered deprecated.  So as projects are updated, we convert them to use `archive` or `release` URLs instead of the default.  Then perhaps at some point, the `github` portgroup's `tarball_from`option can be made to default to `archive`.

More discussion here:
- https://github.com/macports/macports-ports/pull/2587
- https://trac.macports.org/ticket/40518

Reply:
This is still the default in the port group and it works for many ports.

If we want to get rid of `tarball`, I would rather change it in the port group. If there are concerns that it could break ports, then make a github-1.1 port group with the changed default and update the ports. But setting this individually in each port kind of goes against the purpose of a port group in my opinion.

Reply:
Alright, removed.